README.md

```diff
@@ -8,435 +8,429 @@ ACL (Access Control List)
+~~~
+acl { 'c:/tempperms':
+  permissions                => [
+   { identity => 'Administrators', rights => ['full'] },
+   { identity => 'Administrator', rights => ['modify'] },
+   { identity => 'Authenticated Users', rights => ['write','read','execute'] },
+   { identity => 'Users', rights => ['read','execute'] }
+   { identity => 'Everyone', rights => ['read'] }
+  ],
+  inherit_parent_permissions => 'false',
+}
+~~~
+
+ * Each ACE should have a unique combination of `identity`, `perm_type`, `child_types`, and `affects` values. If you create multiple ACEs that differ only in `rights`, the module can't tell them apart and wrongly reports that the resource is out of sync.
+
+
+**Wrong:**
+~~~
+acl { 'c:/tempperms':
+  permissions => [
+    { identity => 'SYSTEM', rights => ['read']},
+    { identity => 'SYSTEM', rights => ['write']}
+  ],
+}
+~~~
 
-The type of target for the ACL resource. The only valid value is 'file'.
+**Right:**
 
-####Properties
+~~~
+acl { 'c:/tempperms':
+  permissions => [
+    { identity => 'SYSTEM', rights => ['read','write']}
+  ],
+}
+~~~
 
-#####`group`
+**Note:** When you run `puppet resource acl some_path`, Puppet might list some permissions with the read-only element `is_inherited => 'true'`. If you use the `resource` output in a manifest, Puppet ignores those permissions. To indicate they should be enforced on the target directly, remove the `is_inherited` property or set `is_inherited => false'`.
 
-The entity or entities that have access to a particular ACL descriptor. The group identity is also known as a trustee or principal. Valid inputs can be in the form of:
+For more detail, see the Reference section on [`permissions`](#permissions).
 
-   * User - e.g. 'Bob' or 'TheNet\Bob'
-   * Group - e.g. 'Administrators' or 'BUILTIN\Administrators'
-   * SID (Security ID) - e.g. 'S-1-5-18'
+###Identify users and groups with SID or FQDN
 
-No default value will be enforced by Puppet. Using the default will allow the group to stay set to whatever it is currently set to (group can vary depending on the original CREATOR GROUP). Since the identity must exist on the system in order to be used, Puppet will make sure they exist by creating them as needed.
+You can identify a user or group using a [security identifier](http://support.microsoft.com/kb/243330) (SID) or a fully qualified domain name (FQDN).
 
-**NOTE**: On Windows the CREATOR GROUP inherited ACE must be set for the creator's primary group to be set as an ACE automatically. Group is not always widely used. By default the group will also need to be specifically set as an explicit managed ACE. See [Microsoft's page](http://support.microsoft.com/kb/126629) for instructions on enabling CREATOR GROUP.
+~~~
+acl { 'c:/tempperms':
+  permissions => [
+   { identity => 'NT AUTHORITY\SYSTEM', rights => ['modify'] },
+   { identity => 'BUILTIN\Users', rights => ['read','execute'] },
+   { identity => 'S-1-5-32-544', rights => ['write','read','execute'] }
+  ],
+}
+~~~
```

Comment:
Maybe it is better to split to two commits, one with the actual changes and one with the reorder.